configuration/microsoft-teams-activity.md

```diff
@@ -4,7 +4,11 @@ description: This article describes how SysKit Point enables you to track Micros
 
 # Enable Microsoft Teams Activity Tracking
 
-**SysKit Point** gives you the possibility to **discover which of your Microsoft Teams are not being actively used**. To collect necessary data, first, SysKit Point Admin has to **send a request to access Protected APIs in Microsoft Teams**. Protected APIs make it possible for SysKit Point to **detect inactive Microsoft Teams based on the date of the last message.**
+**SysKit Point** enables you to easily **discover which of your Microsoft Teams are not being actively used**. 
+
+This provides regular insight on when a workspace should be assessed to determine any possible value for the future with options to **keep, archive or delete the workspace**. These reviews can be **automated** and will be sent to owners after a selected period of time.
```

Comment:
These reviews can be **automated** and will be sent to owners after a selected period of time.
stavio bih rađe nešto tipa 'and will be sent to owners when SysKit Point detects a workspace is inactive' 
U LM postavkama see može definirati 'broj dana neaktivnosti' nakon kojeg se workspace smatra inactive. 
Primjer: taj broj postavimo na 90 dana. Gledamo neki Team. Point svaki dan skuplja Teams aktivnost, SharePoint aktivnost, i Exchange aktivnost za Teamse. Ako prođe 90 dana a za team koji promatramo nemamo ni jednu od tih aktivnosti, šalje se email owneru da je taj Team inactive i da vidi šta želi s njim.

Reply:
* najjednostavnije rečeno - pod Teams aktivnost se misli Teams chat, Exchange aktivnost su mailovi, SharePoint su fileovi i permissioni na njima